tox.ini

```diff
@@ -195,3 +195,10 @@ deps = {[docs]deps}
 setenv = {[docs]setenv}
 commands =
     sphinx-build -W -b linkcheck . {envtmpdir}/linkcheck
+
+[testenv:black]
+deps =
+    black==22.12.0
```

Comment:
We don't need to do it in this PR, but we have black version pinned in 2 places: tox.ini and pre-commit config. It probably makes sense to just fail the CI if `pre-commit run --all-files` changes anything, instead of running these tools in tox manually.